NetKet/Graph/pygraph.hpp

```diff
@@ -182,19 +182,20 @@ void AddGraphModule(py::module& m) {
               Returns the number of vertices in the graph.
            )EOF")
       .def_property_readonly("edges",
```

Comment:
Can we find a compromise here? Maybe you can define another method "edges_iterator" that returns the iterator and edges to return the actual list? NetworkX for `edges` does return a simple list, as far as I can see, and not an iterator, for example

Reply:
I understand that a bit more functional style of programming that I bring may seem too new and even too involved at times, so I won't push it unless I feel it's really important (like in the discussion about Monte Carlo we had about a week ago). So I'll just make `edges` return a list, but do note that `networkx`'s `edges` returns an `EdgeView` rather than a list (at least in version 2.2 on Python 3).

Reply:
OK thanks, let's merge